packages/nimble-components/src/wafer-map/tests/wafer-map.spec.ts

```diff
@@ -116,6 +118,61 @@ describe('WaferMap', () => {
         });
     });
 
+    describe('worker renderer draw wafer action', () => {
```

Comment:
Are we close to parity with unit test coverage for the original and experimental wafer maps now?

Reply:
I think most of the code is covered. There are some drastic changes to the structure of the wafer and a lot of computation is moved in the worker or removed altogether, so I do not think there can be parity. But we repurposed some of the tests and created new tests for the worker. What is left should be covered by the image tests